.travis.yml

```diff
@@ -1,35 +1,29 @@
 language: ruby
-
-rvm: 
-  - 2.5
-
+rvm:
+- 2.5
 env:
   global:
   - CC_TEST_REPORTER_ID=18fcb1674448a0f6f36c0b849ca7bb1f7e9c1fa72688fce866e1add6044cd073
-
-services: 
-  - postgresql
-
+services:
+- postgresql
 before_script:
 - bundle exec rails db:create
 - bundle exec rails db:migrate
-- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
```

Comment:
Think this line + the line below needs to be on one line (like the deleted line above)